.ci/lib.sh

```diff
@@ -322,6 +322,8 @@ gen_clean_arch() {
 		info "delete stale docker resource under ${stale_docker_dir_union[@]}"
 		delete_stale_docker_resource
 	fi
+	info "remove containers started by ctr"
+	sudo sh -c 'ctr c rm $(ctr c ls -q)'
```

Comment:
Please use the 'clean_env_ctr' function instead

Reply:
it is just one line. Do you think we must wrap it?

Reply:
@jongwu, the clean_env_ctr function is already defined, just use it

Reply:
thanks @marcel-apf